models/contact.js

```diff
@@ -0,0 +1,42 @@
+const { Schema, model } = require('mongoose');
+const Joi = require('joi');
+const {handleMongooseError} = require('../helpers');
+
+const contactSchema = new Schema({
+    name: {
+        type: String,
+        required: [true, 'Set name for contact'],
+      },
+      email: {
+        type: String,
+      },
+      phone: {
+        type: String,
+      },
+      favorite: {
+        type: Boolean,
+        default: false,
+      },
+}, {versionKey: false, timestamps: true});
+
+contactSchema.post('save', handleMongooseError);
+
+const addSchema = Joi.object({
+    name: Joi.string().required(),
+    email: Joi.string().required(),
```

Comment:
You should validate email and phone properly, now you could put any strings in here